### PR TITLE
909 - Add action needed and rejected fixtures, add Katherine as analyst

### LINKED ISSUE #909: Add applications to fixtures with status "action needed" and "rejected"
### Story

As a developer or designer
I want to pre-populate our sandboxes with applications that have a status of "action needed" and "rejected"
so that we can test / user test our application with mock data.

### Acceptance Criteria

Loading fixtures on a sandbox will populate it with the usual data plus an application with status "action needed" and another one with status "rejected" for each user.

### Additional Context

_No response_

### Issue Links

_No response_

### PR DESCRIPTION
**Pull request by rachidatecs**
## Ticket

Resolves #909

## Changes

- Add action needed and rejected fixtures
- Add Katherine as analyst

## Setup

@Katherine-Osos, please test your Analyst level access on my sandbox (rjm). Also test your email notifications by changing the status of one of your Analyst user domain applications in a way that triggers an email.

#### Satisfied acceptance criteria and met development standards

- [x] Met the acceptance criteria, or will meet them in a subsequent PR
- [x] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [x] Messaged on Slack or in standup to notify the team that a PR is ready for review

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments

#### Verified that the changes match the design intention

- [ ] Checked behavior

### CODE REVIEW

**Review by neilmb — COMMENTED** [review]
LGTM. It's nice to see that our fixture format is working extensibly.

File: src/registrar/fixtures.py [hunk 1]
```diff
@@ -86,6 +86,12 @@ class UserFixture:
             "first_name": "Alysia-Analyst",
             "last_name": "Alysia-Analyst",
         },
+        {
+            "username": "91a9b97c-bd0a-458d-9823-babfde7ebf44",
+            "first_name": "katherine-Analyst",
```
Comment (zandercymatics) [inline src/registrar/fixtures.py:91]:
```suggestion
            "first_name": "Katherine-Analyst",
```
Was lowercase, not a blocking issue though

Comment (zandercymatics) [inline src/registrar/fixtures.py:91]:
```suggestion
            "first_name": "Katherine-Analyst",
```
This should be upper case

**Review by zandercymatics — APPROVED** [review]
LGTM! I added a suggestion but I won't block you on it, pretty much just a nitpicky thing

**Review by therealslimhsiehdy — APPROVED** [review]

**Comment by rachidatecs** [comment]
Wait, how did it suddenly pass?

**Pull request merged**